workflow/rules/picard_hs_metrics.smk

```diff
@@ -0,0 +1,38 @@
+# vim: syntax=python tabstop=4 expandtab
+# coding: utf-8
+
+__author__ = "Jonas Almlöf"
+__copyright__ = "Copyright 2021, Jonas Almlöf"
+__email__ = "jonas.almlof@scilifelab.uu.se"
+__license__ = "GPL-3"
+
+
+rule picard_hs_metrics:
+    input:
+        bam="alignment/merge_bam/{sample}_{type}.bam",
+        bait_intervals=config["reference"]["design_intervals"],
+        target_intervals=config["reference"]["design_intervals"],
+        reference=config["reference"]["fasta"],
+    output:
+        temp("qc/picard_hs_metrics/{sample}_{type}.HsMetrics.txt"),
+    params:
+        extra=config.get("picard_hs_metrics", {}).get("extra", "COVERAGE_CAP=5000"),
+    log:
+        "qc/picard_hs_metrics/{sample}_{type}.log",
+    benchmark:
+        repeat(
+            "qc/picard_hs_metrics/{sample}_{type}.benchmark.tsv", config.get("picard_hs_metrics", {}).get("benchmark_repeats", 1)
+        )
+    threads: config.get("picard_hs_metrics", {}).get("threads", config["default_resources"]["threads"])
+    resources:
+        threads=config.get("picard_hs_metrics", {}).get("threads", config["default_resources"]["threads"]),
+        time=config.get("picard_hs_metrics", {}).get("time", config["default_resources"]["time"]),
+        mem_mb=config.get("picard_hs_metrics", {}).get("mem_mb", 1024),
```

Comment:
should we have something for this in default_resources? @jonca79 @marrip

Reply:
Definitely!

Reply:
Memory now added to default